tests/common-lib/clusterAccess.js

```diff
@@ -6,31 +6,72 @@ const { execSync } = require('child_process')
 const request = require('supertest')
 const fs = require('fs')
 
-// Login to the cluster
-const clusterLogin = () => {
-  if (!process.env.USE_HUB_KUBECONFIG) {
-    execSync(
-      `oc login -u ${config.get('options:hub:user')} -p ${config.get(
-        'options:hub:password'
-      )} --server=https://api.${config.get(
-        'options:hub:baseDomain'
-      )}:6443 --insecure-skip-tls-verify`
-    )
-  } else {
-    execSync(
-      `oc config use-context --kubeconfig=${process.env.OPTIONS_HUB_KUBECONFIG} ${process.env.OPTIONS_HUB_KUBECONTEXT}`
-    )
+/**
+ * Login into the cluster environment with the `oc` cli command.
+ * @param {object} options Additional options for logging into the cluster environment.
+ */
+const clusterLogin = (options = { useInsecure: true }) => {
+  var cmd = `oc login -u ${config.get('options:hub:user')} -p ${config.get(
+    'options:hub:password'
+  )} --server=https://api.${config.get('options:hub:baseDomain')}:6443`
+
+  if (options.useInsecure) {
+    cmd += ` --insecure-skip-tls-verify`
   }
+
+  execSync(cmd)
```

Comment:
A login error here won't make the test to fail.  That will be problematic to debug later because execution will fail at a later step and it might not be clear that it's because the login failed.  It's better to validate here and throw an error with a clear explanation of why the test failed.

Reply:
Well, this comment is accurate, but I'm more worried about the Cypress path. Adding this comment there.